docs/CONTRIBUTING.md

```diff
@@ -123,17 +122,17 @@ coverage run -m pytest
 coverage report
 ```
 
-And generate a new coverage html for the documentation with
+And generate a new coverage html site for the documentation with
 
 ```bash
 coverage html
 ```
 
 ## Changing dependencies
 
-As the development process moves forward you find you need to add a new
-dependency, just add it to the relevant section of the `pyproject.toml` file
-and recompile requirements:
+As the development process moves forward, you may find you need to add a new
+dependency. Just add it to the relevant section of the `pyproject.toml` file
+and regenerate the requirements files:
 
 ```bash
 pip-compile
```

Comment:
I guess there is no need now even for `requirements.txt`, there's none in the package now, presumably this is something to do with the `hatchling` build in `pyproject.toml`.

```suggestion
dependency. Just add it to the relevant section of the `pyproject.toml` file.
```